nilearn/__init__.py

```diff
@@ -8,67 +8,18 @@
 
 See http://nilearn.github.com for complete documentation.
 """
+import gzip
 
-from version import __version__
+from .version import check_module_dependencies
 
+#
+check_module_dependencies()
+del check_module_dependencies
```

Comment:
I will do that.  I didn't do that because I thought `_check_module_dependencies` would suggest it is private to that module, not shared across files.